README.md

```diff
@@ -201,6 +201,14 @@ The plugin accepts three options:
 
    For a full list of the SVGO config (default) params we are using check out: [svgo-config.js](svgo-config.js). The config you set is merged with our defaults, it does not replace it.
 
+- `fallbackToUseNetworkToLoadImages`: The consumer can point the images to online SVG resources. Example:
```

Comment:
Can we name this variable `allowFromUrl` please? Also, can the default value (`false`) be mentioned on the documentation?
```suggestion
- `allowFromUrl`: defaults to `false`. It allows to use SVG images coming from an URL in addition to local files. For example:
```